docs/plugins/plugin-script.asciidoc

```diff
@@ -279,3 +285,53 @@ The default location of the `plugins` directory depends on which package you ins
 * {ref}/zip-windows.html#windows-layout[Directory layout of Windows `.zip` archives]
 * {ref}/deb.html#deb-layout[Directory layout of Debian package]
 * {ref}/rpm.html#rpm-layout[Directory layout of RPM]
+
+
+[discrete]
+[[manage-plugins-using-configuration-file]]
+=== Managing plugins using a configuration file
+
+[IMPORTANT]
+. Docker only
+=====================
```

Comment:
Not a big deal, but we only require four delimiters. No real reason to use more. For example:

```
[IMPORTANT]
.Docker only
====
My admon text
====
```

Reply:
I was copying from other lines in the file. I'll remember that for the future.